### ISSUE TITLE
Questions about the experiment

### ISSUE THREAD

**Issue by YZGod666** [erdongsanshi/Fuss-Free-structure#5]
As you mentioned in your paper, replacing the PET backbone network with ConvNext_Tiny can have good training effect, but I found that the effect was not satisfactory when I repeated this experiment. I guess the reason may be due to the fine-tuning of parameters. Could you please provide me with the training parameters after replacing the PET backbone network with ConvNEXT_TINY? Thank you very much for your help!

**Comment by erdongsanshi** [comment]
![convnext](https://github.com/user-attachments/assets/ed2bd341-7db6-40db-a8ba-a81d78df40e5)
Hello, as for the PET model replacement experiment, I searched for the previous records, but they were deleted because the memory was cleaned before. I can only provide the replacement structure of the model and the learning rate is 1e-4 or 1e-5 according to experience.

**Comment by YZGod666** [comment]
Ok, thank you very much for your reply. Is the learning rate you reply here the backbone learning rate or the total learning rate?

**Comment by erdongsanshi** [comment]
I think there is no record, I do not know, I just according to my experience to answer, I used more is 1e-4 and 1e-5.

**Comment by YZGod666** [comment]
Add a Upsample to each layer of body in the code you give. This causes the size of each torch under the original torch List feats to change from [2, 96, 64, 64],[2, 192, 32, 32],[2, 384, 16, 16],[2, 768, 8, 8] to [2, 96, 128, 128]. torch.Size([2, 192, 128, 128]), torch.Size([2, 384, 128, 128]), torch.Size([2, 768, 128, 128]), the default feature fusion mode of PET model is feature pyramid network. Different feature sizes are used for feature fusion, which will result in feature fusion errors. What is the meaning of joining Upsample?

**Comment by erdongsanshi** [comment]
According to what you said, I re-tested the previous pet replacement experiment. I used the tensor with the size of (1,3,512,512) as the input, and the final output was shown in the picture, which matched the output size of pet. There was no problem.
![Uploading 1.png…]()

**Comment by erdongsanshi** [comment]
![Uploading 1.png…]()

**Comment by erdongsanshi** [comment]
![1](https://github.com/user-attachments/assets/3bd81352-eedb-4c57-90ce-f1d131ddbab4)
According to what you said, I re-tested the previous pet replacement experiment. I used the tensor with the size of (1,3,512,512) as the input, and the final output was shown in the picture, which matched the output size of pet. There was no problem.
